src/xpk/core/blueprint.py

```diff
@@ -82,7 +105,7 @@ def create_a3_mega_blueprint(
     primary_vpc_name: str = "network1",
     gpu_subnets_name: str = "gpunets",
 ) -> CtkBlueprint:
```

Comment:
Let's have [toolkit_module_version](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/examples/README.md#top-level-parameters) so that upgrades on gcluster version does not require immediate changes on blueprint_generator.

Reply:
not sure what toolkit_module_version should be. A hash? a version?

Reply:
Asked CT team. Let's follow the thread.

Reply:
I think we should skip it for now. It may be added in next PR

Reply:
SGTM. could you create a task for it?

Reply:
done